content/platform/data-services-cloud-console/home.md

```diff
@@ -1,177 +1,42 @@
+To eliminate the silos and complexity of data management and infrastructure, HPE provides the Data Services Cloud Console (DSCC), a SaaS based cloud console that delivers cloud operational agility and unified data operations as a service. Data Services Cloud Console also offers a unified API that gives developers access to infrastructure and data as code. 
 
-![Intelligent Data Platform](/img/dscc-idp-core-architect.png "DSCC Diagram")
+![DSCC UI](/img/a-single-destination-for-data-and-infra-services.png "DSCC UI diagram")
 
-Data Services Cloud Console public REST API provides a resource for customers who are looking to enhance their infrastructure management and data-ops using the programmatical extensions from Data Services Cloud Console. Data Services Cloud Console (DSCC) brings the cloud experience to wherever data lives and streamlines data management across your hybrid cloud. It provides a suite of cloud services across your edge, core, and cloud to accelerate data, agility, and innovation for everyone, from data managers to data innovators.
+Data Services Cloud Console (DSCC) brings the cloud experience to wherever data lives and streamlines data management across your hybrid cloud. It provides a suite of cloud services across your edge, core, and cloud to accelerate data, agility, and innovation for everyone, from data managers to data innovators. Delivered as a SaaS based cloud console, DSCC moved customer’s innovation faster because there is nothing to deploy, always updated to the latest software features, and reduce risk of ownership by subscription. The console is a single destination for any data and infrastructure-services, that comprises of both the cloud data services and cloud infrastructure services.  
 
-![DSCC Automation](/img/dscc-api-value.png "API for Automation at Scale")
+### Introduction to Data Services Cloud Console
 
-## A Public REST API which is based on the OpenAPI 3.0
+To access the Data Service Cloud Console,  click on this URL [](https://common.cloud.hpe.com)<https://common.cloud.hpe.com> and sign-in using your HPE GreenLake Console's account. [](https://common.cloud.hpe.com)
 
-Hewlett Packard Enterprise (HPE) offers the DSCC public REST API to provide the agility as previously mentioned . It is specified based on the OpenAPI format version 3 ([OpenAPI 3.0 information](https://swagger.io/blog/news/whats-new-in-openapi-3-0/)).  The specification defines a standard, language-agnostic interface to the RESTful API allowing the clients (both human and computer) to consume capabilities of DSCC services efficiently. The API definition is available for download in OpenAPI 3 YAML or JSON format at the link mentioned in the next paragraph.
+For an  overview of the Data Services Cloud Console and to get started with the Data services Cloud Console please click on this link <https://hpe.com/storage/dscc>. 
```

Comment:
Recommend rephrasing to: "For an overview of the Data Services Cloud Console and to get started using it, please click on this link."